### PR TITLE
Android Wifi Manager

### PR DESCRIPTION
**Pull request by mwoz123**
https://github.com/koreader/android-luajit-launcher/pull/65 enables android wifi management.
This is Koreader implemention part that make use of above

Not tested. Testers welcome.

### CODE REVIEW

File: frontend/device/android/device.lua [hunk 1]
```diff
@@ -51,4 +51,16 @@ function Device:init()
...
+    NetworkMgr.turnOnWifi = function()
+        android.setWifiEnabled(true)
+    end
+
+    NetworkMgr.turnOffWifi = function()
+        android.setWifiEnabled(false)
+    end
+end
+
+
```
Comment (Frenzie) [inline frontend/device/android/device.lua]: Stray newlines.

**Comment by Frenzie** [comment]
Works well. You also need to change this line to `enabled_func = function() return Device:isAndroid() or Device:isKindle() or Device:isKobo() end,` to be able to disable it again.

https://github.com/koreader/koreader/blob/199b6aba51790ee4b687e6bdd7175edfcfd9150e/frontend/ui/network/manager.lua#L85

File: frontend/device/android/device.lua (hunk 1 above)
Comment (mwoz123) [reply frontend/device/android/device.lua]: removed but does it have any influence to the app? Or is just a Koreadernon-sense

**Comment by mwoz123** [comment]
> Works well. You also need to change this line to enabled_func = function() return Device:isAndroid() or Device:isKindle() or Device:isKobo() end, to be able to disable it again.

Overlooked that. Thanks

**Comment by Frenzie** [comment]
Thanks!

**Pull request merged**